Dockerfile

```diff
@@ -1,34 +1,57 @@
-#
+### ### ### ### ### ### ### ### ###
 # Builder layer
-#
+
 FROM node:6-alpine as ghost-builder
-RUN npm install --loglevel=error -g ghost-cli
 
-ENV GHOST_VERSION 1.17.1
-ENV GHOST_INSTALL /var/lib/ghost
-ENV GHOST_CONTENT /var/lib/ghost/content
-ENV GHOST_USER node
+RUN \
+    apk update && apk upgrade                           && \
+    echo                                                && \
+    echo "--- Install ghost-cli --- "; echo             && \
+    npm install --loglevel=error -g ghost-cli           ;
+
+ENV GHOST_VERSION="1.17.1"                              \
+    GHOST_INSTALL="/var/lib/ghost"                      \
+    GHOST_CONTENT="/var/lib/ghost/content"              \
+    GHOST_USER="node"
 
+# Set default directory
 WORKDIR $GHOST_INSTALL
 
 # Run SQLite as database
 RUN \
-	ghost install "$GHOST_VERSION" --db sqlite3 --no-prompt --no-stack --no-setup --dir "$GHOST_INSTALL"; \
-	ghost config --ip 0.0.0.0 --port 2368 --no-prompt --db sqlite3 --url http://localhost:2368 --dbpath "$GHOST_CONTENT/data/ghost.db"; \
-	ghost config paths.contentPath "$GHOST_CONTENT";
+    ghost install "$GHOST_VERSION" \
+        --db sqlite3 --no-prompt \
+        --no-stack --no-setup \
+        --dir "$GHOST_INSTALL"                          && \
+    ghost config --ip 0.0.0.0 \
+        --port 2368 --no-prompt --db sqlite3 \
+        --url http://localhost:2368 \
+        --dbpath "$GHOST_CONTENT/data/ghost.db"         && \
+    ghost config paths.contentPath "$GHOST_CONTENT"     ;
 
 COPY run-ghost.sh $GHOST_INSTALL
 
-#
+# Here we could add custom themes within the Docker image
+
+# Keeping Original GhostContent to be copied into the mounted volume (if empty)
+RUN cp -r "$GHOST_CONTENT" "$GHOST_INSTALL/content.bck" ;
+
+
+### ### ### ### ### ### ### ### ###
 # Final image
-#
+# No tzdata as it's not working on alpine3.4 (from node6)
+
 FROM node:6-alpine
 LABEL maintainer="Marco Mornati <marco@mornati.net>"
 
-ENV GHOST_VERSION 1.17.1
-ENV GHOST_INSTALL /var/lib/ghost
-ENV GHOST_CONTENT /var/lib/ghost/content
-ENV GHOST_USER node
+RUN apk update && apk upgrade                           && \
```

Comment:
I think it is not a best practice to execute an update and upgrade in a docker. The packages are managed by the base Docker. Do you really need to upgrade them for your one?
And without these two commands you don't even need to remove the cache folder